test/gtest/ucp/test_ucp_am.cc

```diff
@@ -1014,7 +1014,7 @@ UCS_TEST_P(test_ucp_am_nbx_eager_data_release, single_zcopy, "ZCOPY_THRESH=0")
     test_data_release(fragment_size() / 2);
 }
 
-UCS_TEST_SKIP_COND_P(test_ucp_am_nbx_eager_data_release, multi, enable_proto())
+UCS_TEST_P(test_ucp_am_nbx_eager_data_release, multi)
```

Comment:
i think better to rename it to `multi_bcopy` and set "ZCOPY_THRESH=0", so you can later add bcopy